sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp

```diff
@@ -10,6 +10,9 @@
 // REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
 
+// https://github.com/intel/llvm/issues/14826
```

Comment:
I suggest updating this instead: "REQUIRES-INTEL-DRIVER: lin: **30049**, win: 101.4943"